README.zh.md

```diff
@@ -1,159 +1,75 @@
-
-    ```bash
-    cd RTOS
-    rustup override set beta-2021-06-23-x86_64-unknown-linux-gnu
-    ```
-
-    添加Rust的`no-std`组件。
-
-    ```bash
-    rustup component add rust-src
-    ```
-  
-3. 选择编译选项
-
-    创建默认配置：
-
-    ```bash
-    export CROSS_COMPILE=aarch64-linux-gnu-
-    export ARCH=arm64
-
-    make LLVM=1 defconfig
-    make LLVM=1 menuconfig
-    ```
-
-    选择以下选项：
-
-    ```
-    General Setup --->  Rust Support
-    Kernel Features ---> Bupt real-time core
-    ```
-
-    您可能需要取消选项`Module versioning support`以启用`Rust support`：
-
-    ```
-    Enable loadable module support ---> Module versioning support
-    ```
-
-4. 编译内核
-
-    ```bash
-    make LLVM=1 -j
-    ```
-
-    如果您想在Raspiberry PI 4上启动，请额外生成dtbs和modules。
-
-    ```bash
-    export INSTALL_MOD_PATH=/path/to/mod
-    export INSTALL_DTBS_PATH=/path/to/dtbs
-    make modules_install dtbs_install -j
-    ```
-
-    将`broadcom`，`lib`，`overlays`和`Image`移动到SD卡的引导分区。
-
-5. 在模拟器上运行
-
-    您需要一个文件系统来在QEMU上引导内核。
-
-    这是一个在qemu上运行的示例：
-
-    ```bash
-    qemu-system-aarch64 -nographic  \
-          -kernel Image \
-          -drive file=ubuntu-20.04-server-cloudimg-arm64.img \
-          -drive file=cloud_init.img,format=raw \
-          -initrd ubuntu-20.04-server-cloudimg-arm64-initrd-generic \
-          -machine virt-2.12,accel=kvm \
-          -cpu host  -enable-kvm \
-          -append "root=/dev/vda1 console=ttyAMA0"  \
-          -device virtio-scsi-device \
-          -smp 4 \
-          -m 4096
-    ```
+[RROS极速入门手册](https://bupt-os.github.io/website/docs/introduction/quick-start/)：如何启动、运行、测试和开发 RROS。
 
 ## 文档
 
-详细信息可以参考我们的[文档](https://bupt-os.github.io/website/docs/)。
+可以从这里查看我们的[文档](https://bupt-os.github.io/website/docs/)。
 
-## 贡献
+## 联系方式与贡献方法
 
-如果想要贡献代码，可以参考[这里](https://bupt-os.github.io/website/docs/contributing/contributing/)。
+您可以通过 [Zulip 论坛](https://rros.zulipchat.com/) 或电子邮件 `buptrros AT gmail.com` 与我们联系。
 
-## 路线图
-
-查看[这里](https://bupt-os.github.io/website/docs/roadmap/roadmap/)了解我们未来的路线图
+我们非常欢迎大家共建社区！详情请看[贡献指南](https://bupt-os.github.io/website/docs/contributing/contributing/)。
 
-## 联系我们
+## 路线图
 
-- 论坛：您还可以在[Zulip](https://rros.zulipchat.com/)上与我们联系。
-- 电子邮件：我们的电子邮箱是`buptrros AT gmail.com`。
+查看[路线图](https://bupt-os.github.io/website/docs/roadmap/roadmap)以了解我们未来的规划。
 
-## 发布
+## 我们是谁
 
-RROS依赖于dovetail和RFL，但是它们目前不再给出补丁。而将它们中的一个高频率下地回合到另一个是非常困难的。 因此，RROS目前固定在5.13版本的Linux内核上，它是在linux-dovetail-v5.13的基础上构建的，并且很容易打RFL patch v1的补丁。 幸运的是，RFL正在快速合并到Linux内核主线中。我们计划在我们所依赖的大多数RFL API被合并到linux-dovetail的主线后开始以固定频率发布版本。 届时，我们将一并选择LTS版本。
+我们是北京邮电大学的一个[研究小组](https://bupt-os.github.io/website/docs/team/team/)。
 
-## 团队
+## 发行版
 
-查看[这里](https://bupt-os.github.io/website/docs/team/team/)了解我们的团队
+RROS 依赖于 dovetail 和 Rust for Linux(RFL)，目前两者都没有提供补丁。将两者最新的代码高频率地回合到一个项目非常具有挑战性。因此，RROS 暂时被锁定在 Linux 内核版本 5.13 上，这是基于 linux-dovetail-v5.13 构建的，这个版本的linux与 RFL 补丁 v1 兼容。同时幸运的是，RFL 正在迅速进入主线 Linux 内核。我们计划当我们依赖的大部分 RFL API 被 linux-dovetail 主线后，我们将切换新的dovetail版本。届时，我们将进一步考虑长期支持（LTS）。

```

Comment:
In the penultimate sentence of this paragraph, it should be merged into `linux-dovetail`. The sentence is unclear.

Reply:
Fixed.